src/styles/stars.css

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  padding-top: 17px;
```

Comment:
why did you add padding-top for the stars block? let's add correct margins for the card__content element